core/platform/auth/firebase_auth_services.py

```diff
@@ -408,6 +411,8 @@ def _firebase_admin_context():
         logging.exception('Firebase Admin SDK raised an exception!')
     finally:
         if app is not None:
+            # NOTE: This is not dangerous. This is how the Firebase SDK cleans
```

Comment:
Can you maybe explain what "app" means here? Is the variable name appropriate?

Reply:
Done.